src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,45 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class Lotto {
+    private static final int LOTTO_NUMBER_SIZE = 6;
+    private static final String NOT_CORRECT_LOTTO_NUMBER_SIZE_MESSAGE = "6개의 수를 입력하셔야 합니다.";
```

Comment:
```suggestion
    private static final String NOT_CORRECT_LOTTO_NUMBER_SIZE_MESSAGE = LOTTO_NUMBER_SIZE +"개의 수를 입력하셔야 합니다.";
```

기존에 있던 LOTTO_NUMBER_SIZE을 활용해서 메시지를 구성해도 좋을것 같아요:)